### ISSUE TITLE
Readtimeout error message popups - not user friendly

### ISSUE THREAD

**Issue by madhurrya** [sartography/spiff-arena#1443]
I lost my internet connection for about 10-15 minutes and after that most of the spiff tabs are showing this popup. 

![Image](https://github.com/sartography/spiff-arena/assets/110527567/6bad0b1f-a6d0-4dc4-ab2c-fb54f3948ba3)

Some times that popup doesn't close at first time and has to keep closing it.

It's better to show a more user friendly message when no internet or for other similar conditions.

**Comment by tcoz** [comment]
Yes, it's pretty straightforward. Please talk to your new UI dev about it.
